### PR TITLE
Fix miscellaneous things from contrib migration PR

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Addresses the PR review comments from https://github.com/purescript-contrib/purescript-colors/pull/50#pullrequestreview-918821869 and add `purs-tidy` to CI.

---

**Checklist:**

- [ ] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
- [x] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation in the README and/or documentation directory
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by JordanMartinez** [comment]
:ping_pong: @thomashoneyman

File: src/Color.purs [hunk 1]
```diff
@@ -108,17 +108,21 @@ data ColorSpace = RGB | HSL | LCh | Lab
 
 instance showColor :: Show Color where
   show c = "rgba " <> show col.r <> " "
```
Comment (thomashoneyman) [inline src/Color.purs:110]: Maybe:

```
show c = String.joinWith " "
  [ "rgba"
  , show col.r
  , show col.g
  , show col.b
  , show col.a
  ]

File: src/Color.purs [hunk 2]
```diff
@@ -108,17 +108,21 @@ data ColorSpace = RGB | HSL | LCh | Lab
...
+    <> show col.a
+    where
+    col = toRGBA c
 
 instance eqColor :: Eq Color where
-  eq c1 c2 = rgb1.r == rgb2.r && rgb1.g == rgb2.g &&
-             rgb1.b == rgb2.b && rgb1.a == rgb2.a
+  eq c1 c2 = rgb1.r == rgb2.r && rgb1.g == rgb2.g
+    && rgb1.b == rgb2.b
+    && rgb1.a == rgb2.a
```
Comment (thomashoneyman) [inline src/Color.purs:122]: Perhaps:

```purs
eq = eq `on` \c -> let { r, g, b, a } = toRGBA c in [ r, g, b, a ]
```

**Review by thomashoneyman — APPROVED** [review]

**Comment by JordanMartinez** [comment]
I'm waiting to merge this until CI runs again, too.

**Pull request merged**